### DISCUSSION TITLE
need help!!

### DISCUSSION THREAD

**Discussion by Damji7** [python-telegram-bot/python-telegram-bot#4037] [Q&A]
am making script i insert this code:

       def start_bot(self, bot_token):
        updater = Updater(bot_token, use_context=True)
        dispatcher = updater.dispatcher   

        dp.add_handler(AdminHandler(admin_ids))
        dp.add_handler(CommandHandler("start", self.start))
        dp.add_handler(CommandHandler("quit", self.quit))
        dp.add_handler(CommandHandler("help", self.help))        

        updater.start_polling()
        updater.idle() " 

so when i run my script it shows me this "  updater = Updater(bot_token, use_context=True)
                                                                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: Updater.__init__() got an unexpected keyword argument 'use_context'  " 

i did everything from changing the codes to uninstalling and installin telegram bot and downgrade upgrades.... etc. so just am in problem anyone have solution guys ??

**Comment by Poolitzer** [comment]
Hey. You seem to be using a version >=20.0 of python-telegram-bot  but your code is written for version <=13.15. Please either upgrade your code to  v20 by reading the [release notes](https://telegra.ph/Release-notes-for-python-telegram-bot-v200a0-05-06) and the [transition guide](https://github.com/python-telegram-bot/python-telegram-bot/wiki/Transition-guide-to-Version-20.0) or install a version of PTB that is compatible with your code base.
Please note that we only provide support for the latest stable version and that the library has undergone significant changes in v20.
